proto/palomachain/paloma/evm/user_smart_contract.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package palomachain.paloma.evm;
+
+import "cosmos_proto/cosmos.proto";
+
+option go_package = "github.com/palomachain/paloma/x/evm/types";
+
+enum DeploymentStatus {
+  DEPLOYMENT_STATUS_UNSPECIFIED = 0;
+  DEPLOYMENT_STATUS_ONGOING = 1;
```

Comment:
We might want to keep it more in line with the existing lingo on smart contract deployments for this state. Take a look at their status names, I think it's like `PENDING` or `IN_FLIGHT`.